src/main/java/com/algolia/search/SearchConfig.java

```diff
@@ -13,7 +13,21 @@
 
     /** Builds a {@link SearchConfig} with the default hosts */
     public Builder(@Nonnull String applicationID, @Nonnull String apiKey) {
-      super(applicationID, apiKey);
+      super(applicationID, apiKey, createDefaultHosts(applicationID, apiKey));
+    }
+
+    @Override
+    public Builder getThis() {
+      return this;
+    }
+
+    public SearchConfig build() {
+      return new SearchConfig(this);
+    }
+
+    /** Create default hosts for the search configuration */
+    private static List<StatefulHost> createDefaultHosts(
+        @Nonnull String applicationID, @Nonnull String apiKey) {
```

Comment:
here too: `applicationID` and `apiKey` are unused, I suggest to do without

Reply:
Fixed with eb0e845 :)

Reply:
Hmmmm are you sure? I am still seeing it in the final diff